packages/aws-cdk/lib/api/aws-auth/sdk-provider.ts

```diff
@@ -273,19 +282,19 @@ export class SdkProvider {
   public async defaultAccount(): Promise<Account | undefined> {
     return cached(this, CACHED_ACCOUNT, async () => {
       try {
-        return await new SDK(this.defaultCredentialProvider, this.defaultRegion, this.requestHandler, this.logger).currentAccount();
+        return await new SDK(this.defaultCredentialProvider, this.defaultRegion, this.requestHandler, this.ioHelper, this.logger).currentAccount();
       } catch (e: any) {
         // Treat 'ExpiredToken' specially. This is a common situation that people may find themselves in, and
         // they are complaining about if we fail 'cdk synth' on them. We loudly complain in order to show that
         // the current situation is probably undesirable, but we don't fail.
         if (e.name === 'ExpiredToken') {
-          warning(
+          await this.ioHelper.notify(IO.DEFAULT_SDK_WARN.msg(
```

Comment:
I think I might quibble with this naming.

This is not actually a warning "from" the SDK, it's a warning from our SDK wrapper.

Reply:
I think my argument is that these are all prefixed `CDK_SDK` and we don't claim them to be from the SDK directly. Is there an actual benefit in separating these out?